MIGRATING.md

```diff
@@ -0,0 +1,65 @@
+# Migrating
+
+This guide lists API changes between releases of *PoE* contracts.
+
+## 0.6.0-beta1 -> 0.6.0-beta2
```

Comment:
Nice guide. Thanks.

Minor point: I would label the next step 0.6.0-rc1 (with all the breaking api changes), not beta2